src/components/InspectorExtensionManager.js

```diff
@@ -30,7 +30,16 @@ function addInspectorItem(inspectorItems, config) {
 
 function getInspectorItems(node, config) {
   if (typeof config.container !== 'undefined') {
+    translateConfig(node.inspectorConfig[0].items);
     return node.inspectorConfig[0].items;
   }
   return node.inspectorConfig[0].items[0].items;
```

Comment:
Why does this method need to be called in `registerInspectorExtension` at all? It seems to me that a client of `getInspectorItems` would expect to have an array of items returned that are populated with translations, rather than having to call another function to add these as an extra step. Could you move this change to be inside `getInspectorItems` instead?

Reply:
What @JCook21 said would be a better approach, something like:

```js
function getInspectorItems(node, config) {
  const inspectorItems = typeof config.container === 'undefined'
    ? node.inspectorConfig[0].items[0].items
    : node.inspectorConfig[0].items;

  return translateConfig(inspectorItems);
}
```

This would still cover the additional case when `config.container` is defined.